compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/ManifestBuilder.java

```diff
@@ -170,13 +171,10 @@ private PackageManifest parseAsPackageManifest() {
         }
 
         // Do not mutate toml tree
-        Map<String, TopLevelNode> otherEntries = new HashMap<>();
+        Map<String, Object> otherEntries = new HashMap<>();
         if (!tomlAstNode.entries().isEmpty()) {
-
-            for (Map.Entry<String, TopLevelNode> entry : tomlAstNode.entries().entrySet()) {
-                if (entry.getKey().equals(PACKAGE)) {
-                    continue;
-                }
+            Map<String, Object> tomlMap = ballerinaToml.toml().toMap();
+            for (Map.Entry<String, Object> entry : tomlMap.entrySet()) {
```

Comment:
Does otherEntries contain [Package] table as well? is that the expected behavior?

Reply:
@hevayo `[package]` table also can have other entries as we discussed earlier. Should we avoid that?

Reply:
`[package]` and `[platform]` are kept as reserved and these fields won't be added to the other entries

Reply:
Great :)